locksmithctl/daemon.go

```diff
@@ -352,5 +372,14 @@ func runDaemon() int {
 	close(stop)
 	wg.Wait()
 
+	if window != nil {
+		fmt.Printf("Waiting for reboot window %q\n", window)
```

Comment:
this will print even if there's no need to wait...  We should only print this on the first entry of the loop below.